src/MIDebugPackage/MIDebugPackage.csproj

```diff
@@ -268,6 +268,7 @@
     <DropSignedFile Include="$(OutDir)\Microsoft.MICore.XmlSerializers.dll" />
     <DropSignedFile Include="$(OutDir)\Microsoft.MIDebugEngine.dll" />
     <DropUnsignedFile Include="$(OutDir)\Microsoft.MIDebugEngine.pdb" />
+    <DropUnsignedFile Include="$(OutDir)\Microsoft.VisualStudio.Debugger.Interop.15.0.dll" />
```

Comment:
><DropUnsignedFile Include="$(OutDir)\Microsoft.VisualStudio.Debugger.Interop.15.0.dll" /> [](start = 4, length = 89)

Will this wind up in the CoreXT package we insert into VS? If so, do we want to add a condition here to exclude it from the MDD build?